api/octokit.ts

```diff
@@ -243,38 +253,103 @@ export class OctoKit implements GitHub {
 		}
 	}
 
-	async getProjectNodeId(projectId: number, org: string): Promise<string | undefined> {
-		console.debug('Running getProjectNodeId for project ' + projectId)
+	async getProject(
+		projectId: number,
+		org: string,
+		columnName?: string,
+	): Promise<ProjectAndColumnIds | undefined> {
+		console.debug('Running getProject for project ' + projectId)
 
 		try {
 			const result = (await this._octokitGraphQL({
 				query: `query getProjectNodeId($org: String!, $projectId: Int!) {
-				organization(login:$org){
-				  projectNext(number:$projectId) {
+					organization(login: $org) {
+					  projectNext(number: $projectId) {
 						id
+						fields(first: 100) {
+						  edges {
+							node {
+							  id
+							  name
+							}
+						  }
+						}
 					  }
-				}
-				}`,
+					  project(number: $projectId) {
+						id
+						state
+						columns(first: 100) {
+						  edges {
+							node {
+							  id
+							  name
+							}
+						  }
+						}
+					  }
+					}
+				  }
+				`,
 				projectId: Math.floor(projectId),
 				org,
 			})) as GraphQlQueryResponseData
-			console.debug('getProjectNodeId result ' + JSON.stringify(result))
-			return result.organization.projectNext?.id
+			console.debug('getProject result ' + JSON.stringify(result))
+
+			if (result.organization.projectNext && result.organization.projectNext.id) {
+				return {
+					projectType: projectType.ProjectNext,
+					projectNodeId: result.organization.projectNext.id,
+				}
+			} else if (result.organization.project && result.organization.project.id && columnName) {
+				// try to find the right column
+				for (const column of result.organization.project.columns.edges) {
+					if (column.node.name === columnName) {
+						return {
+							projectType: projectType.Project,
+							projectNodeId: result.organization.project.id,
+							columnNodeId: column.node.id,
+						}
+					}
+				}
+
+				return undefined
+			}
 		} catch (error) {
 			console.error('Could not get project node id ' + error)
 		}
 		return undefined
 	}
 
-	async addIssueToProject(projectId: number, issue: Issue, org = 'grafana'): Promise<void> {
-		console.debug('Running addIssueToProject for: ' + projectId)
-
-		const projectNodeId = await this.getProjectNodeId(projectId, org)
+	protected async addIssueToProjectOld(projectColumnId: string, issueNodeId: string) {
+		console.log(
+			'Running addIssueToProjectOld with: projectColumnId: ',
+			projectColumnId,
+			' issueNodeId: ',
+			issueNodeId,
+		)
+		const mutation = `mutation addProjectCard($projectColumnId: String!, $issueNodeId: String!) {				
+			addProjectCard(input: {projectColumnId: $projectColumnId, contentId: $issueNodeId) {
```

Comment:
@marefr yes, old projects have a concept of [cards](https://docs.github.com/en/rest/reference/projects#create-a-project-card) and cards can be associated with issues. You have to add card to a column when you add card to a project.

Reply:
Okay. But from GitHub UI you can assign issue/PR to old project and it will go "needs triage" (no column) like in this example 
![image](https://user-images.githubusercontent.com/1668778/145622954-1a9d132a-4d73-436a-b386-5d7e894af967.png)

If you have automation setup you can define that an issue/PR should be placed in certain column when added to a project.